gci/models.py

```diff
@@ -1 +1,117 @@
-# Create your models here.
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+from django.db import models
+
+
+class Student(models.Model):
+
+    identifier = models.IntegerField(primary_key=True)
+    display_name = models.CharField(max_length=100)
+    program_year = models.ManyToManyField('ProgramYear')
+
```

Comment:
shouldn't there be an email field for student as well? like in mentor.

Reply:
its not in the data we get